src/traces/contour/close_boundaries.js

```diff
@@ -32,36 +34,31 @@ module.exports = function(pathinfo, operation, perimeter, trace) {
         boundaryMax = Math.max(boundaryMax, z[nb - 1][i]);
     }
 
+    pi0.prefixBoundary = false;
+
     switch(operation) {
         case '>':
-        case '>=':
-            if(trace.contours.value > boundaryMax) {
-                pathinfo[0].prefixBoundary = true;
+            if(contours.value > boundaryMax) {
+                pi0.prefixBoundary = true;
             }
             break;
         case '<':
-        case '<=':
-            if(trace.contours.value < boundaryMin) {
-                pathinfo[0].prefixBoundary = true;
+            if(contours.value < boundaryMin) {
+                pi0.prefixBoundary = true;
             }
             break;
         case '[]':
-        case '()':
-            v1 = Math.min.apply(null, trace.contours.value);
-            v2 = Math.max.apply(null, trace.contours.value);
-            if(v2 < boundaryMin) {
-                pathinfo[0].prefixBoundary = true;
-            }
-            if(v1 > boundaryMax) {
-                pathinfo[0].prefixBoundary = true;
+            v1 = Math.min.apply(null, contours.value);
+            v2 = Math.max.apply(null, contours.value);
+            if(v2 < boundaryMin || v1 > boundaryMax) {
+                pi0.prefixBoundary = true;
             }
             break;
         case '][':
-        case ')(':
-            v1 = Math.min.apply(null, trace.contours.value);
-            v2 = Math.max.apply(null, trace.contours.value);
+            v1 = Math.min.apply(null, contours.value);
+            v2 = Math.max.apply(null, contours.value);
```

Comment:
Shouldn't we use `Lib.aggNums` here in case `contours.value` has some non-numeric values?

Reply:
`handleConstraintValueDefaults` already cleans this up. Not sure if I actually *agree* with all the choices made there, but it doesn't look to me as though non-numeric values can get through.